Sources/EmbraceCore/Public/Embrace+CrashHelper.swift

```diff
@@ -18,4 +50,26 @@ extension Embrace {
         CrashHelper.crash(example: type)
     }
 
+    /// Appends additional key-value information to the next crash report.
+    ///
+    /// This method allows the addition of a key-value pair as an attribute to the next occurring
+    /// crash that is reported during the lifetime of the process. This can be useful for adding context
+    /// or debugging information that may help in analyzing the crash when exported.
+    ///
+    /// - Important: Throws an exception if no proper `CrashReporter` is configured.
+    ///
+    /// - Parameters:
+    ///   - key: The key for the attribute.
+    ///   - value: The value associated with the given key.
+    @objc public func appendCrashInfo(key: String, value: String) throws {
```

Comment:
Would pull this and the error class out into a different file to separate it from the integration helper `crash(type:)`.